scribejava-apis/src/test/java/com/github/scribejava/apis/examples/TTNV1Example.java

```diff
@@ -0,0 +1,102 @@
+package com.github.scribejava.apis.examples;
+
+import com.github.scribejava.apis.TheThingsNetworkV1StagingApi;
+import com.github.scribejava.core.builder.ServiceBuilder;
+import com.github.scribejava.core.model.OAuth2AccessToken;
+import com.github.scribejava.core.model.OAuthRequest;
+import com.github.scribejava.core.model.Response;
+import com.github.scribejava.core.model.Verb;
+import com.github.scribejava.core.oauth.OAuth20Service;
+
+import java.io.IOException;
+import java.net.URLDecoder;
+import java.util.Random;
+import java.util.Scanner;
+
+public final class TTNV1Example {
```

Comment:
should be named like Api class.
It's hard to find corresponding Example otherwise